targets/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # Add all targets
-add_subdirectory(blinkUtil)
-add_subdirectory(BIKE_NAME-BOARD_NAME)
+add_subdirectory(REV3-WSS)
+add_subdirectory(hallSensor)
```

Comment:
Technically, this sample name should use snake_case, but it doesn't really matter